README.rst

```diff
@@ -32,7 +46,28 @@ Known not to work:
 * remove/upgrade: not handled yet
 * update: slow and solution often subobtimal.
 
-Bibliography:
+Comparing with php's composer
+=============================
+
+First, clone composer's somewhere on your machine::
+
+    git clone https://github.com/composer/composer
+
+Then, use the `scripts/scenario_to_php.py` script to write a php file that will
+print the composer's solution for a given scenario::
+
+    python --composer-root <path to composer github checkout> \
```

Comment:
The `--composer-root` option apparently needs to come after the `scenario_to_php` script, and that script is a `.py` script, not a `.php` script.